src/containers/Experiment/SamplesTable.js

```diff
@@ -68,25 +57,10 @@ export default class SamplesTable extends React.Component {
                     selectedOption={this.state.pageSize}
                     onChange={this.handlePageSizeChange}
                   />
-                  of {samples.length} Samples
+                  of {this.totalSamples} Samples
```

Comment:
In reference to my above comment, so you wouldn't need to do `this.totalSamples()`?